docs/plugins/development/git.md

```diff
@@ -60,25 +60,129 @@ This plugin will significantly slow down the speed of data preparation, especial
 
 ### contributors
 
-- Type: `boolean`
+- Type: `boolean | ContributorsOptions`
+
+  ```ts
+  interface ContributorsOptions {
+    /**
+     * Functions to transform contributors, e.g. remove duplicates ones and sort them.
+     * The input is the contributors collected by this plugin, and the output should be the transformed contributors.
+     */
+    transform?: (
+      contributors: GitContributor[],
+    ) => GitContributor[] | Promise<GitContributor[]>
+
+    /**
+     * List of contributors configurations
+     */
+    list?: ContributorConfig[]
+
+    /**
+     * Whether to add avatar in contributor information
+     * @default false
+     */
+    avatar?: boolean
+  }
+
+  interface ContributorConfig {
+    /**
+     * Contributor's username on the git hosting service
+     */
+    username: string
+    /**
+     * The alias of the contributor,
+     * Since contributors may have different usernames saved in their local git configuration
+     * compared to their usernames on the hosting service, In this case, aliases can be used to
+     * map to the actual usernames.
+     */
+    alias?: string[] | string
```

Comment:
I think that providing a hook and letting users to make their own merge comparison is better

Reply:
The plugin still retains `contributors.transform` for handling `contributors`. This configuration mainly provides a simple way to preset contributor information.